python/pmtiles/v2.py

```diff
@@ -1,3 +1,4 @@
+import json
```

Comment:
Unrelated, but also necessary

Reply:
@fscottfoti the JS version is backwards compatible with v2; do you have a need for this for the python client as well, instead of converting your archives from v2 to v3 with https://github.com/protomaps/go-pmtiles ?

For some context, I don't have an internal use case for the Python library ATM since the Lambda implementation at https://github.com/protomaps/PMTiles/tree/main/serverless/aws changed from Python to Node for v3. So if there is 0 use cases for v2 reading in Python I'd prefer to remove this module completely.

Reply:
Not really a use case but it was handy to be able to upgrade *before* upgrading all the files.  I was appreciative it was there.  If you decide to keep it you could always keep the single most recent version or something but it's not required.

Thanks for keeping the python API without a personal use case.  It's much appreciated.